dipy/tracking/utils.py

```diff
@@ -919,37 +789,29 @@ def unique_rows(in_array, dtype='f4'):
 
 
 @_with_initialize
-def move_streamlines(streamlines, output_space, input_space=None,
-                     seeds=None):
+def transform_tracking_output(tracking_output, affine, save_seeds=False):
```

Comment:
I think this function should be deleted in favor of `transform_streamlines`. You can add the seed management inside `transform_streamlines`

Reply:
As mentioned in the other comment, transforming streamlines is confusing enough, I think this would make the `transform_streamlines` function less clear by making it doing too many thing at once